library/os.po

```diff
@@ -3926,7 +3926,6 @@ msgid "Create a symbolic link pointing to *src* named *dst*."
 msgstr "Crée un lien symbolique pointant vers *src* et appelé *dst*."
 
 #: ../Doc/library/os.rst:2694
-#, fuzzy
```

Comment:
Il manque `If the target is present` dans la traduction, mais il y a le "Sinon" de la cible absente, c'est louche.